smsjwplatform/acl.py

```diff
@@ -0,0 +1,182 @@
+"""
+Module providing lookup API-related functionality.
```

Comment:
ACL related functionality, surely? Also, I can't see this module mentioned in ``doc/smsjwplatform.rst``. I suggest that the lookup functionality be moved into a ``lookup.py`` module. (If it is identical to the ``lookup.py`` module we use elsewhere then it makes it easier to keep an eye on moving this module into a separate repo.)